aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: kubernetes-dashboard
-version: 2.0.1
+version: 2.0.1-1
```

Comment:
Not sure why we haven't started chart versioning from the `1.0.0`. Just bump the minor version to `2.0.2`.

Reply:
Actually, this was an experiment to keep the versions synchronized and release the chart at the same time as Dashboard. There are cons and pros, I'm open to suggestions.

Reply:
IIRC in order to use the new chart features version has to be bumped, otherwise, helm yells at you that chart version didn't change, right? If so, keeping both versions in sync would mean that in order to use those new chart features people would have to wait for official Dashboard release.

Reply:
That is the "con" part, indeed. I do not mind having separated versions!

Reply:
OK, I have changed the version to `2.1.0`.

Regarding the mismatch of the app and helm version, I have originally chosen absolutely valid version `2.0.1-1` which maintains the relationship between the two versions but still increments it by specifying the revision portion. I think that's quite a nice compromise.

Reply:
After some thoughts according to your inputs, I think un-coupling the versions is the best way to go and the only one to realistically respect semver. 2.0.1-1 is not semver-compliant, and waiting for a new minor release to release this is not realistic either, so 2.1.0 seems the way to go!